src/runtime/clrobject.cs

```diff
@@ -14,7 +14,7 @@ internal CLRObject(object ob, IntPtr tp)
             System.Diagnostics.Debug.Assert(tp != IntPtr.Zero);
             IntPtr py = Runtime.PyType_GenericAlloc(tp, 0);
 
-            long flags = Util.ReadCLong(tp, TypeOffset.tp_flags);
+            var flags = (TypeFlags)Marshal.ReadInt32(tp, TypeOffset.tp_flags);
```

Comment:
`tp_flags` is a `long`, not an `int32`.

Reply:
Reverted back to using `Util.ReadCLong`.
Surprisingly in `PyType_Spec` flags are of type `unsigned int`, which implies they should never exceed `2^32`.

Reply:
Hm. Actually, because of the above we can probably use `ReadInt32`.
`tp_flags` is the only field, that is assumed to be declared as `long`. So we could remove `ReadCLong` and `WriteCLong` helpers then.